Tarea de calculo de nota

MiguelMoraissaimai opened the pull request.

MiguelMoraissaimai committed:
paso dos

MiguelMoraissaimai committed:
paso tres

MiguelMoraissaimai committed:
ultimo commit

MiguelMoraissaimai committed:
Merge branch 'mmasias:main' into main

MiguelMoraissaimai committed:
cambios desde web

MiguelMoraissaimai committed:
Merge branch 'mmasias:main' into main

Commit:
java primer commit

Commit:
ultimo commit de arreglo

MiguelMoraissaimai committed:
si

Commit:
commit tras hacer pruebas de texto e integers

Commit:
calculo de nota hecho

Commit:
division añadida entre ejerecicio inicial de prueba de int, double, etc y el ejercicio de las notas

Commit:
ejercicio terminado y arhchivos finalizados

Commit:
arreglo por olvido de archivo eliminado

Commit:
arreglo por olvido de archivo eliminado... otra vez

Commit:
Merge branch 'main' of https://github.com/MiguelMoraissaimai/24-25-PRG1

mmasias committed:
Update aDiaDeHoy.md

mmasias commented on entregas/moraisMiguel/java/CalculaNotaFinal.java:
```diff
@@ -0,0 +1,35 @@
+
+
+
+        /** valor de notas:
+         * examen parcial  25%
+         * examen final 50%
+         * evaluacion del profesor 5%
+         * evaluacion continua 20%
+         */
```
No use comentarios

mmasias commented on entregas/moraisMiguel/java/CalculaNotaFinal.java:
```diff
@@ -0,0 +1,35 @@
+
+        double mediaFinal;
+
+        mediaFinal = examenFinal * 0.5 + examenParcial * 0.25 + evaluacionContinua * 0.2 + evaluacionProfesor * 0.05;
```
Siempre es mejor EXAMEN_FINAL_PONDERACION que un 0.5: aporta más información en el cálculo.

mmasias commented on entregas/moraisMiguel/java/HolaMundo.java:
```diff
@@ -0,0 +1,45 @@
+
+        dineroTrasBitcoin = dineroNuevo + 100000000;
+
+        /** Intente hacer exponente pero dado que es un double y no un int me dio error */
```
Fíjate: estas explicaciones que se suelen poner en los comentarios quedan mucho  mejor en el commit que haces de tu código o incluso en un issue. De ese modo, cuando lo corrijas no hace falta que estés eliminando los comentarios: recuérdamelo en clase porfa...

mmasias commented:
👍🏼

Commit:
mates added

mmasias committed:
Update aDiaDeHoy.md

MiguelMoraissaimai committed:
Merge branch 'mmasias:main' into main

MiguelMoraissaimai committed:
generador de numeros aleatorios

mmasias committed:
Actualización de retos y detalles de entrega

- Eliminado el texto genérico sobre la entrega de retos y sus fechas límite.
- Actualizada la tabla en el README de retos con información específica de entrega para el Reto 001 y el nuevo Reto 002, incluyendo fechas de entrega y ramas específicas.
- Añadido el archivo reto002.md con la estructura inicial y la fecha de entrega especificada.

mmasias committed:
fix: Titulo

mmasias committed:
Dos ejemplos...

mmasias committed:
Tres ejemplos ternarios

mmasias committed:
Enunciado de los ejemplos

mmasias committed:
Enlace en la página de ejercicios

mmasias committed:
Incluye salida esperada en contenido de ejercicios

mmasias committed:
Update ClasificacionEdad.md

MiguelMoraissaimai committed:
Merge branch 'mmasias:main' into main

mmasias committed:
Definición de reto 001

mmasias committed:
Refinamos requisito (Human vs Computer)

mmasias committed:
Una imagen...

MiguelMoraissaimai committed:
Merge branch 'mmasias:main' into main

Commit:
piedraPapelTijera añadido

Commit:
Merge branch 'main' of https://github.com/MiguelMoraissaimai/24-25-PRG1

MiguelMoraissaimai committed:
arreglos a piedra papel tijera

MiguelMoraissaimai committed:
reto 2 terminado, puesto en entregas como 'piedraPapelTijera.java'

Commit:
añadido juegos for

Commit:
added caracol

Commit:
updated caracol

Commit:
akjdnas

MiguelMoraissaimai committed:
derrota

mmasias committed:
Código!

mmasias committed:
Caracol ternario!

mmasias committed:
Update aDiaDeHoy.md

mmasias committed:
Update README.md

mmasias committed:
Recopilación y ordenado de ejercicios

mmasias committed:
Create .gitignore

mmasias committed:
Ejercicios++

mmasias committed:
Update README.md

mmasias committed:
🪨📄✂️🦎🖖🏻

mmasias committed:
Update aDiaDeHoy.md

mmasias committed:
Un dia en la vida de Marco
Pendiente:
- La madre
- Los dias

Commit:
marco added

Commit:
Merge branch 'main' of https://github.com/MiguelMoraissaimai/24-25-PRG1

mmasias committed:
Solucion hecha en clase

mmasias committed:
Update ViajeMarco.java

Es más lógico expresarlo así:

seHanEncontrado si la distancia entre Marco y su madre es menor o igual a cero.

mmasias committed:
Propuesta de solución fullTernario 💪🏼💪🏼💪🏼

mmasias committed:
Actualización con soluciones

mmasias committed:
CarreraCamellos hecho en clase

mmasias committed:
Publicada la solución propuesta de CarreraCamellos

mmasias committed:
Extensión

Commit:
Carrear

Commit:
batalla

mmasias committed:
Avance de clase. Inicio de solución de la Batalla

MiguelMoraissaimai committed:
Merge branch 'mmasias:main' into main

mmasias committed:
Update README.md

mmasias committed:
Versión minimalista
Vídeo de construcción en https://www.youtube.com/watch?v=bebuAZLYAVE

mmasias committed:
Update README.md

mmasias committed:
Ejercicio del centro comercial
Además incluyo una corrección en el ejercicio de la carrera de camellos.

mmasias committed:
Solución (hecha en clase) del centro comercial

mmasias committed:
Fix & version ASCII

mmasias committed:
Update README.md

mmasias committed:
Fix, inclusión de hora/minuto y animación

mmasias committed:
Segundo intento

mmasias committed:
3rd

mmasias committed:
Enlaces

mmasias committed:
Centro comercial resuelto con métodos estáticos
No he puesto commits individuales :( Disculparme

MiguelMoraissaimai committed:
intento de juego de adivinar el numero (me complique demasiado)

MiguelMoraissaimai committed:
Merge branch 'main' of https://github.com/MiguelMoraissaimai/24-25-PRG1

MiguelMoraissaimai committed:
Merge branch 'mmasias:main' into main

MiguelMoraissaimai committed:
matrices

MiguelMoraissaimai committed:
Merge branch 'main' of https://github.com/MiguelMoraissaimai/24-25-PRG1

MiguelMoraissaimai committed:
mas practica de matrices

MiguelMoraissaimai committed:
practicas

MiguelMoraissaimai committed:
juego del topo

MiguelMoraissaimai committed:
alquimia comienzo

MiguelMoraissaimai committed:
viajemarco

MiguelMoraissaimai committed:
string equality
